Neos.Flow/Classes/Validation/Validator/GenericObjectValidator.php

```diff
@@ -43,6 +48,15 @@ public function setValidatedInstancesContainer(\SplObjectStorage $validatedInsta
         $this->validatedInstancesContainer = $validatedInstancesContainer;
     }
 
+    /**
+     * @param object $object The object instance to register a validation callback for
+     * @param \Closure $callback The callback to invoke once this object has been validated
+     */
+    public static function onValidated($object, \Closure $callback): void
```

Comment:
This should become `object $object` in 6.0/PHP 7.2

Reply:
That means _now_, by now. 😎

Reply:
```suggestion
    public static function onValidated(object $object, \Closure $callback): void
```